src/SFA.DAS.ApplyService.Database/Script.PostDeployment1.sql

```diff
@@ -11,24 +11,20 @@ Post-Deployment Script Template
 */
 
 -- cleanup (for now)
-DELETE FROM [EmailTemplates]
+BEGIN
+	DELETE FROM EmailTemplates;
```

Comment:
im being a bit of a wussy, but a blanket delete makes me nervous.  Can we add 'WHERE TEMPLATEID in ('RoATPGetHelpWithQuestion','RoATPApplicationSubmitted',etc?)